### PR TITLE
Merge qa to master: Update README for Cloudhosted guides (#121)

### PR DESCRIPTION
**Pull request by gkwan-ibm**
* Update README.adoc

* Update README.adoc

* Update README.adoc

* Update README.adoc

* Update README.adoc

* Update README.adoc

* Update README.adoc

* Update README.adoc

* Update README.adoc

* Update README.adoc

* Update README.adoc

* Update README.adoc

* Update README.adoc

Co-authored-by: Gilbert Kwan <gkwan@ca.ibm.com>

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -46,7 +46,7 @@ to directly convert JSON messages into Java objects. The second approach consist
 using the Java API for JSON Processing (JSON-P) to process the JSON.
 
 The REST service that provides the artists and albums resources has already been written
```
Comment (dmuelle) [inline README.adoc:48]: Global questionfor clarification: are values such as `http://localhost:9080/artists` and `http://localhost:9080/artists/total` endpoints or URLs? These terms are both used but I think it should be one term consistently, unless the examples are fundamentally different, in which case it should be clear how they are different.

My understanding is that something like `/artists/total/<artist>` is an endpoint, while `http://localhost:9080/artists/total/<artist>` is a URL, which includes the endpoint, but also other information for the HTTP address. Is this correct?

**Review by dmuelle — COMMENTED** [review]
quick question before I review, see comment

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc:48]: yes, `http://localhost:9080/artists/total/<artist>` is an URL and `/artists/total/<artist>` is an endpoint in this guide.

If just read from an URL itself, it cannot 100% tell what is the endpoint. e.g. `http://localhost:9080/a/b/c`.  endpoint can be either
- `/a/b/c` if the app context root is `/`
- `/b/c` if the app context root is `/a`
- `/c` if the app context root is `/a/b`

Comment (dmuelle) [reply README.adoc:48]: ok, if the endpoint is the part of the URL we want to call attention to, can consistently refer to them as endpoints

File: README.adoc [hunk 2]
```diff
@@ -46,7 +46,7 @@ to directly convert JSON messages into Java objects. The second approach consist
...
-for you and is accessible at the following link when the server is running http://localhost:9080/artists[http://localhost:9080/artists^], which responds with the [hotspot]`artists.json`.
+for you and is accessible at the following link when the server is running `\http://localhost:9080/artists`, which responds with the [hotspot]`artists.json`.
```
Comment (dmuelle) [inline README.adoc:49]: suggested edit:
The REST service that provides the artists and albums resources has already been written for you and is accessible at the following link when the server is running http://localhost:9080/artists, which responds with the artists.json.
--->
The REST service that provides the artist and album resources is already written for you. When the server is running, this service  is accessible at the  http://localhost:9080/artists endpoint, which responds with the artists.json file.

if artists.json  is a file? or is it and object?

File: README.adoc [hunk 3]
```diff
@@ -68,9 +68,37 @@ include::{common-includes}/gitclone.adoc[]
 [role='command']
 include::{common-includes}/twyb-intro.adoc[]
 
-You can find your service at http://localhost:9080/artists.
+// static guide instructions:
+ifndef::cloud-hosted[]
+You can find your service at http://localhost:9080/artists[http://localhost:9080/artists^].
```
Comment (dmuelle) [inline README.adoc:73]: You can find your service at the http://localhost:9080/artists[http://localhost:9080/artists ^] endpoint.

File: README.adoc [hunk 4]
```diff
@@ -68,9 +68,37 @@ include::{common-includes}/gitclone.adoc[]
...
+
+Now, you can access the endpoint at http://localhost:9080/artists/total[http://localhost:9080/artists/total^] to see the total number of artists, and you can access the endpoint at `\http://localhost:9080/artists/total/<artist>` to see a particular artist’s total number of albums.
+
+endif::[]
+
+// cloud-hosted guide instructions:
+ifdef::cloud-hosted[]
+Open another command-line session by selecting **Terminal** > **New Terminal** from the menu of the IDE.
+
+You can find your service at **http://localhost:9080/artists** by running the following curl command:
```
Comment (dmuelle) [inline README.adoc:83]: You can find your service at  the **http://localhost:9080/artists** endpoint by running the following curl command:

File: README.adoc [hunk 5]
```diff
@@ -276,10 +329,35 @@ party service, you should separate your `GET`/`POST` requests from your data con
...
+ifndef::cloud-hosted[]
 You can find your service at http://localhost:9080/artists[http://localhost:9080/artists^].
 
 Now, you can access the endpoint at http://localhost:9080/artists/total[http://localhost:9080/artists/total^] to see the total number of artists,
 and you can access the endpoint at `\http://localhost:9080/artists/total/<artist>` to see a particular artist's total number of albums. 
+endif::[]
+
+// cloud-hosted guide instructions:
+ifdef::cloud-hosted[]
+You can find your service at **http://localhost:9080/artists** by running the following curl command:
```
Comment (dmuelle) [inline README.adoc:342]: You can find your service at the **http://localhost:9080/artists** endpoint by running the following curl command

**Review by dmuelle — COMMENTED** [review]
see my comments inline, lmk if any questions

File: README.adoc (hunk 2 above)
Comment (gkwan-ibm) [reply README.adoc:49]: should be file

**Review by dmuelle — APPROVED** [review]
LGTM

**Pull request merged**